en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -128,16 +128,22 @@ This minimum data set is required for all EKF modes of operation. Other sensor d
 ### Magnetometer
 
 Three axis body fixed magnetometer data (or external vision system pose data) at a minimum rate of 5Hz is required.
+
 Magnetometer data can be used in two ways:
 
-* Magnetometer measurements are converted to a yaw angle using the tilt estimate and magnetic declination.
-  This yaw angle is then used as an observation by the EKF.
-  This method is less accurate and does not allow for learning of body frame field offsets, however it is more robust to magnetic anomalies and large start-up gyro biases.
-  It is the default method used during start-up and on ground.
-* The XYZ magnetometer readings are used as separate observations.
-  This method is more accurate and allows body frame offsets to be learned, but assumes the earth magnetic field environment only changes slowly and performs less well when there are significant external magnetic anomalies.
+- Magnetometer measurements are converted to a yaw angle using the tilt estimate and magnetic declination.
+  The yaw angle is then used as an observation by the EKF.
+  - This method is less accurate and does not allow for learning of body frame field offsets, however it is more robust to magnetic anomalies and large start-up gyro biases.
+  - It is the default method used during start-up and on ground.
+- The XYZ magnetometer readings are used as separate observations.
+  - This method is more accurate when the vehicle is moving, as it allows body frame offsets to be learned (when the vehicle is moving the heading or the magnetometer biases are observable).
+    The method is less robust, and the accuracy benefits are not available when the vehicle is not moving.
```

Comment:
Thanks @bresch . I'll steal all your words and make this more precise then. 
I'm going to merge, but would appreciate a final check when you have a moment.

```suggestion
  - This method is more accurate but requires that the magnetometer biases are correctly estimated.
    - The biases are observable while the drone is rotating and the true heading is observable when the vehicle is accelerating (linear acceleration).
    - Since the biases can change and are only observable when moving, it is safer to switch back to heading fusion when not moving.
```